### PR TITLE
fix(statutory): only allow applications from valid bodies

### PR DESCRIPTION
**Pull request by LeonVreling**
Fixes [MA-123](https://myaegee.atlassian.net/jira/software/c/projects/MA/issues/MA-123), and also implements that you auto-select your primary body, and not just the first body in the list

Should be merged together with [statutory-#422](https://github.com/AEGEE/statutory/pull/422)

### CODE REVIEW

File: src/views/statutory/EditApplication.vue [hunk 1]
```diff
@@ -497,11 +497,19 @@ export default {
       // If this is a new application, then allowing selecting bodies which the current user is a member of
       // and not fetching application.
       if (this.isNew) {
-        this.bodies = this.loginUser.bodies
+        for (const body in this.loginUser.bodies) {
```
Comment (serge1peshcoff) [inline src/views/statutory/EditApplication.vue:500]: this one can be simplified as

`this.bodies = this.loginUser.bodies.filter(body => this.can.apply_from_body[body.id])`

**Review by serge1peshcoff — APPROVED** [review]

File: src/views/statutory/EditApplication.vue (hunk 1 above)
Comment (LeonVreling) [reply src/views/statutory/EditApplication.vue]: Yes will update it

**Comment by serge1peshcoff** [comment]
@LeonVreling don't forget to rename this PR to match commits schema

**Comment by WikiRik** [comment]
Woops, I just did that

**Comment by WikiRik** [comment]
Okay, so this did not work 100%. It only works when you create the application, but not when you edit it. @LeonVreling will make a new PR to fix this

**Pull request merged**